changelog.d/8153.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in v1.19.0 that would cause non-join operations (e.g. calling `/join` on a room the user has already joined, or sending a profile update) to be rate-limited.
```

Comment:
Could we frame this in terms of not being able to set display names? Reading that doesn't make me think its much of a problem. Something like the following maybe?

```suggestion
Fix a bug introduced in v1.19.0 that would cause e.g. profile updates to fail due to incorrect application of rate limits on join requests.
```